NineChronicles.Headless.Executable/Program.cs

```diff
@@ -242,9 +242,7 @@ static void ConfigureSentryOptions(SentryOptions o)
                 //o.Debug = true;
                 o.Release = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>()
                     ?.InformationalVersion ?? "Unknown";
-                o.SampleRate = headlessConfig.SentryTraceSampleRate > 0
-                    ? (float)headlessConfig.SentryTraceSampleRate
-                    : 0.01f;
+                o.SampleRate = 0.01f;
                 o.TracesSampleRate = headlessConfig.SentryTraceSampleRate;
```

Comment:
@moreal that option is still used, but only for trace

Reply:
Ah, it is my mistake. Thank you 🙏🏻